src/plugins/filter/filter.style.scss

```
@@ -112,19 +114,20 @@ revogr-filter-panel .filter-panel-dialog {
     border-radius: 4px;
 
     &.active {
-      color: #10224a;
+      color: var(--rg-theme-filter-panel-icon-active);
     }
 
     .filter-img {
       $img-size: 11px;
-      color: gray;
+      color: var(--rg-theme-filter-panel-icon);
```

Stylelint (17.14.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)